types/plugin.d.ts

```diff
@@ -1,6 +1,12 @@
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
This is due to the optionnal type : `Options extends FastifyPluginOptions = {}`.

Reply:
So may be we can replace it with `Options extends FastifyPluginOptions = FastifyPluginOptions` ?